tensorbay/label/subcatalog.py

```diff
@@ -301,7 +300,10 @@ def add_keypoints(
             number: The number of keypoints.
             names: All the names of keypoints.
             skeleton: The skeleton of keypoints.
-            visible: The visible type of keypoints.
+            visible: The visible type of the keypoints, can only be 'BINARY' or 'TERNARY'.
+                It determines the range of the
+                :attr:`Keypoint2D.v<tensorbay.geometry.keypoint.Keypoint2D.v>`.
+                See :class:`~tensorbay.geometry.keypoint.Keypoint2D` for details.
```

Comment:
just remove the details here, if visible has details, other arguments like skeleton should have details too.